src/compat/gt-130/ArkoudaBitOpsCompat.chpl

```diff
@@ -0,0 +1,7 @@
+module ArkoudaBitOpsCompat {
+  use BitOps;
+
+  proc popCount(x: integral) {
+    return popcount(x);
+  }
+}
```

Comment:
For greater than 1.30, we have the `popCount` method in Chapel I think, so we don't need this overload (i.e., we need the `ArkoudaBitOpsCompat` module for all 3 versions of Chapel, but the one that supports the deprecated feature, we can just have it be an empty module, like https://github.com/Bears-R-Us/arkouda/blob/master/src/compat/gt-130/ArkoudaBigIntCompat.chpl).

Does that make sense? Also possible that I am missing something.

Reply:
Yeah that makes sense, I'll remove that function.